react/PartnerSites/Sites/Sites.js

```diff
@@ -79,7 +98,12 @@ export default function PartnerSites({ locale, linkRenderer }) {
   );
 }
 
-PartnerSites.propTypes = {
-  locale: PropTypes.string.isRequired,
-  linkRenderer: PropTypes.func.isRequired
+Sites.propTypes = {
+  locale: PropTypes.oneOf(['AU', 'NZ']).isRequired,
+  linkRenderer: PropTypes.func.isRequired,
```

Comment:
This prop should be optional, I think.

Reply:
We just need to provide a default `linkRenderer`, like we do with `Header`.

Reply:
This is a child/private component so it should be required from my understanding. I believe the PartnerSites props are optional as per convention.